lib/instrumentation/http-shared.js

```diff
@@ -90,6 +95,8 @@ exports.traceOutgoingRequest = function (agent, moduleName) {
         agent.logger.debug('request details: %o', {protocol: protocol, host: req._headers.host, id: id})
       }
 
+      ins.bindEmitter(req)
```

Comment:
Should we bind the client response as well?

Reply:
Probably, yes. I meant to mark this as in-progress. Just wanted to get your input on it first to make sure there wasn't something I was missing in regard to not wrapping emitter handlers before.